### PR TITLE
body: add crc and counter

### PR DESCRIPTION
**Pull request by briskspirit**
Add crc and counter to important CAN messages.

### CODE REVIEW

File: selfdrive/car/body/bodycan.py [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+from selfdrive.car import crc8_pedal
+
+
+def create_control(packer, torque_l, torque_r, idx):
   can_bus = 0
 
   values = {
     "TORQUE_L": torque_l,
     "TORQUE_R": torque_r,
+    "COUNTER": idx & 0xF,
```
Comment (adeebshihadeh) [inline selfdrive/car/body/bodycan.py:10]: packer should set this

Comment (briskspirit) [reply selfdrive/car/body/bodycan.py:10]: Why do we do the same way for pedal then? in create_gas_interceptor_command()

Comment (briskspirit) [reply selfdrive/car/body/bodycan.py:10]: Also because of the signal name?

Comment (adeebshihadeh) [reply selfdrive/car/body/bodycan.py:10]: not sure, sounds likely though

**Pull request merged**